formulaire/src/main/resources/public/ts/controllers/form-editor.ts

```diff
@@ -170,9 +170,11 @@ export const formEditorController = ng.controller('FormEditorController', ['$sco
                 }
             }
 
-            // Check choice.image and choice.value for questions of type Types.MULTIPLEANSWER
-            let questionTypeMultipleanswers: Question[] = vm.formElements.getAllQuestions().filter((q: Question) => q.question_type == Types.MULTIPLEANSWER);
-            const foundChoice = (<any>questionTypeMultipleanswers).flatMap((question: Question) => question.choices.all)
+            // Check choice.image and choice.value for questions of type MULTIPLEANSWER & SINGLEANSWER
+            let multipleanswerOrSingleanswer: Question[] =
+                vm.formElements.getAllQuestions().filter((q: Question) => (q.question_type == Types.MULTIPLEANSWER)
+                    || (q.question_type == Types.SINGLEANSWERRADIO));
```

Comment:
Vu que tu fais 3x q.question_type == Types.MULTIPLEANSWER || q.question_type == Types.SINGLEANSWERRADIO, tu devrais généraliser ça dans une fonction dans le model Question.ts et le réutiliser ici, du genre :

    canHaveImages = () : boolean => {
        return this.question_type == Types.SINGLEANSWER
            || this.question_type == Types.SINGLEANSWERRADIO;
    }

```suggestion
                vm.formElements.getAllQuestions().filter((q: Question) => q.canHaveImages());
```